src/Build/Utilities/FileSpecMatchTester.cs

```diff
@@ -42,7 +43,7 @@ public bool IsMatch(string fileToMatch)
                 return _regex.IsMatch(normalizedFileToMatch);
```

Comment:
Shouldn't we account for ignoring case here as well?

Reply:
Absolutely. The beautiful (terrible?) part is that it already ignored case:
https://github.com/dotnet/msbuild/blob/master/src/Shared/FileMatcher.cs#L51

I can add a test for that, though.